langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -124,7 +133,8 @@ public AzureOpenAiChatModel(String endpoint,
                                 List<AzureChatExtensionConfiguration> dataSources,
                                 AzureChatEnhancementConfiguration enhancements,
                                 Long seed,
-                                ChatCompletionsResponseFormat responseFormat,
```

Comment:
same comments as above for this and other ctors

Reply:
same, reverted to the former API